### PR TITLE
feat(route): web.dev

### LINKED ISSUE #14728: new-to-the-web  更新内容抓取
### 类型

编程

### 网站地址

https://web.dev/series/new-to-the-web?hl=zh-cn

### 网站描述

一篇月度博文，概括介绍浏览器引入的一些新事物。

### 需要生成什么内容？

抓取标题和内容

### 额外描述

_No response_

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

### PR DESCRIPTION
**Pull request by KarasuShin**
## Involved Issue / 该 PR 相关 Issue

Close #14728

## Example for the Proposed Route(s) / 路由地址示例

```routes
/web/blog
/web/articles
/web/series/new-to-the-web
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/web/series.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import type { Data, Route } from '@/types';
+import { fetchItems, hyphen2Pascal } from './utils';
+
+export const route: Route = {
+    path: '/series/:seriesName',
```
Comment (TonyRL) [inline lib/routes/web/series.ts:5]: Missing `parameters` property to explain `seriesName`
docs: https://docs.rsshub.app/joinus/new-rss/start-code#creating-route

Comment (KarasuShin) [reply lib/routes/web/series.ts:5]: thanks for your review

File: lib/routes/web/articles.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: {
+        source: ['web.dev/articles'],
+        target: '/web/articles',
+    },
```
Comment (TonyRL) [inline lib/routes/web/articles.ts:19]:
```suggestion
    radar: [{
        source: ['web.dev/articles'],
    }],
```

File: lib/routes/web/blog.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: {
+        source: ['web.dev/blog'],
+        target: '/web/blog',
+    },
+    name: 'Blog',
```
Comment (TonyRL) [inline lib/routes/web/blog.ts:20]:
```suggestion
    radar: [{
        source: ['web.dev/blog'],
    }],
    name: 'Blog',
```

File: lib/routes/web/series.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: {
+        source: ['web.dev/series/:seriesName'],
+        target: '/web/articles/:seriesName',
+    },
```
Comment (TonyRL) [inline lib/routes/web/series.ts:20]:
```suggestion
    radar: [{
        source: ['web.dev/series/:seriesName'],
        target: '/series/:seriesName',
    }],
```

**Pull request merged**